.github/workflows/static.yml

```diff
@@ -9,17 +9,16 @@ on:
   # Allows you to run this workflow manually from the Actions tab
```

Comment:
I think that there is a problem on line 7. there is no $default-branch as a variable on the github. needs to be updated.

our ci/cd process will be as follows, can you review again the code, check something is missing or not. 

`-> build -> run the tests -> PR(code-review) -> approval -> merge into master -> deploy to github pages`